scout/commands/update/disease.py

```diff
@@ -16,7 +16,7 @@
 from flask.cli import current_app, with_appcontext
 
 from scout.constants import UPDATE_DISEASES_RESOURCES
-from scout.load.hpo import load_disease_terms
```

Comment:
scout.load.hpo was also loading OMIM terms, so I've renamed it to avoid confusion

Reply:
If you check the code it should already be split, lines 54 and 60, no?